crypto/armcap.c

```diff
@@ -12,6 +12,7 @@
 #include <string.h>
 #include <setjmp.h>
 #include <signal.h>
+#include <sys/param.h>
```

Comment:
I think we need to keep the `#include <sys/param.h>` conditional, presumably also on `defined(__FreeBSD__)`, since that's not a terribly portable header.  (I'm actually rather surprised that the CI was happy with it.)